lib/robot.js

```diff
@@ -241,4 +241,9 @@ module.exports = (...args) => new Robot(...args);
   * robot.log.warn("Woah there");
   * robot.log.error("ETOOMANYLOGS");
   * robot.log.fatal("Goodbye, cruel world!");
+  *
+  * Note: Probot supports execption tracking through raven, a client for
+  * [sentry](https://github.com/getsentry/sentry). If the `SENTRY_DSN` is set
+  * as an environment varialbe, all errors will be forwarded to the sentry host
+  * specified by the environment variable.
```

Comment:
> …all errors will be forwarded to the sentry host…

Is it any error passed to `robot.log`, or only those passed to `robot.log.error` (I haven't actually tested it)? Either way, might be good to specify that here.